cone.js

```diff
@@ -1,17 +1,18 @@
 "use strict";
 
-const V = require('gl-vec3');
-const V4 = require('gl-vec4');
+var V = require('gl-vec3');
```

Comment:
cc @kig plotly.js src code must be es5 compliant. You can use a es6->es5 browserify transform if you prefer in your other gl-vis modules.